packaging/domain-transfer/merged_assembly/create_stimulus_set_with_background-id.py

```diff
@@ -0,0 +1,34 @@
+# import libraries 
+## general libraries 
+import pandas as pd
+
+## brain-score libraries
+import brainscore
+
+## domain-transfer specific libraries
+from helpers_background_id import loading_brain_model_activation
+
+
+# get the background-id
+brain_model_name='dcgan'
+image_source='dicarlo.domain_transfer'
+
+domain_transfer_data = loading_brain_model_activation(brain_model_name, image_source, penultimate_layer=False)
+
+background_df = pd.DataFrame({'id': domain_transfer_data.stimulus_id.values, 'background': domain_transfer_data.background_id.values})
```

Comment:
you don't need a model to achieve this, the `background_id` is in the stimulus set

Reply:
That would be right if I were to define the background indices from scratch. However, for the sake of time, I used Marlia's code, in which she defined `background_id` as a `presentation` dimension of the results of the `look_at` function, which needs a model (any model).

Reply:
the `presentation` coordinates in `look_at` come exclusively from the stimulus set, so the background id needs to already be present there